dz12_ex3.py

```diff
@@ -0,0 +1,23 @@
+#3. В попередньо написаний кастомний Exception додати запис помилки і час її виникнення у файл.
+
+import time
+
+class MyCustomException():
```

Comment:
I don't know why you decided to create a context manager here, but the tasks was to implement custom exception class with logging to a file. I gave you almost full solution in my previous comment. Here are a bit more:
```
class MyCustomException(Exception):
    def __init__(self, message):
        with open('file', 'a') as file:
            file.write(message)

```
That's it. Add here date and time and change filename and it will be completed

Reply:
@ISirota I still don't see a custom exception in your solution. You implemented `CONTEXT MANAGER` and you need to implement `CUSTOM EXCEPTION` 

My example above is 99% of the whole task. Just copy it to your code, change filename and add timestamp to log